src/modules/mavlink/mavlink_log_handler.h

```diff
@@ -105,15 +65,43 @@ class MavlinkLogHandler
 	unsigned get_size();
 
 private:
+	enum class LOG_HANDLER_STATE {
+		INACTIVE,     //There is no active action of log handler
```

Comment:
Can you change the naming convention:
```suggestion
	enum class LogHandlerState {
		Inactive,     //There is no active action of log handler
```